src/libraries/System.Net.Http.Json/ref/System.Net.Http.Json.csproj

```diff
@@ -0,0 +1,12 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
```

Comment:
This needs to be multi-targeted for full framework to avoid bringing nestandard shims when used in full framework, right, @joperezr ?

Reply:
I haven't pushed @joperezr's changes for multi-target.